homework/new_coffee/coffee.py

```diff
@@ -23,16 +23,16 @@ def prepare_time_coffee(self):
         time_to_prepare = self.water/10 + (self.coffee_spoons*10 + self.sugar*5)/5
```

Comment:
Line 9: quantity should be defined as a local variable within a method, not global. Otherwise when instantiating it multiple times it will add up to quantity